lib/activity/pull-requests.js

```diff
@@ -3,17 +3,23 @@ const { PullRequest } = require('../messages/pull-request');
 const cache = require('../cache');
 
 async function pullRequestEvent(context, subscription, slack) {
-  const eventType = `${context.event}.${context.payload.action}`;
-  // need seperate api request to get labels and body_html
-  const issue = (await context.github.issues.get(context.issue({
-    headers: { accept: 'application/vnd.github.html+json' },
-  }))).data;
-
-  const pullRequest = {
+  let pullRequest = {
     ...context.payload.pull_request,
-    labels: issue.labels,
-    body_html: issue.body_html,
   };
+  if (context.payload.action === 'opened') {
+    // need seperate api request to get labels and body_html
+    const issue = (await context.github.issues.get(context.issue({
+      headers: { accept: 'application/vnd.github.html+json' },
+    }))).data;
```

Comment:
Why only `opened`? What happens if I edit a PR?

Reply:
We don't actually do anything at the moment when you edit a PR. That will be implemented in https://github.com/integrations/slack/issues/335

We only route the following to this function:
https://github.com/integrations/slack/blob/master/lib/activity/index.js#L36